reference/5.1/Microsoft.PowerShell.Core/About/about_Classes.md

```diff
@@ -452,19 +452,19 @@ Devices                       Brand          Model
 Notice **Slots** property isn't shown in `$r1` output. However, the size was
 changed by the constructor.
 
-## Static attribute
+## Static keyword
 
-The `static` attribute defines a property or a method that exists in the class
+The `static` keyword defines a property or a method that exists in the class
 and needs no instance.
 
 A static property is always available, independent of class instantiation. A
 static property is shared across all instances of the class. A static method is
 available always. All static properties live for the entire session span.
 
-### Example using static attributes and methods
+### Example using static keywords and methods
```

Comment:
I think this is more accurately about static _properties_ and methods, using the static keyword.

```suggestion
### Example using static properties and methods
```